docs_src/pipelinerun.md

```diff
@@ -1,128 +1,78 @@
+ 
+ The following table lists the commands for various input combinations. Run the command based on your requirement.
 
-!!! success
-    Your output is as follows:
-
-| IMAGE                                              | STATUS                   | NAMES             |
-| -------------------------------------------------- | ------------------------ |-------------------|
-| openvino/ubuntu20_data_runtime:2021.4.2            | Up 11 seconds            | simulator_docker  |
-| aler9/rtsp-simple-server                           | Up 13 seconds            | camera-simulator  |
-
-!!!Note there could be multiple containers with IMAGE "openvino/ubuntu20_data_runtime:2021.4.2", depending on number of sample-media video file you have.
-
-!!! failure
-    If you do not see all of the above docker containers, look through the console output for errors. Sometimes dependencies fail to resolve and must be run again. Address obvious issues. To try again, repeat [Run camera simulator](#run-camera-simulator).
-
----
-## Run pipeline with different input source(inputsrc) types
-Use docker-run.sh to run the pipeline, here is the table of basic scripts for each combination:
-
-| Input source Type | scripts                                                                                                |
+| Input Source Type | Scripts                                                                                                |
 | ----------------- | -------------------------------------------------------------------------------------------------------|
-| simulated camera  | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc rtsp://127.0.0.1:8554/camera_0</code>|
-| realsense camera  | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc <serial_number> --realsense_enabled</code>        |
+| Simulated camera  | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc rtsp://127.0.0.1:8554/camera_0</code>|
+| RealSense camera  | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc <serial_number> --realsense_enabled</code>        |
 | USB camera        | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc /dev/video0</code>                         |
-| a video file      | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc file:my_video_file.mp4</code>             |
+| Video file      | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc file:my_video_file.mp4</code>             |
+    
 
-!!!Note for `dgpu.x`, the x can be 0, 1, 2, ... depends on how many discrete gpu it has in the system
-
----
-## Optional parameters
-The optional parameters you can apply to the docker-run.sh input, just note that they will affect the performance of pipeline run.
+**_Note:_**  The value of x in `dgpu.x` can be 0, 1, 2, and so on depending on the number of discrete GPUs in the system.
+    
```

Comment:
**_Note:_** Here is [How_to_query_usb_camera](./query_usb_camera.md)

Reply:
What is the reason to query USB camera? Adding the explanation would make it clearer.

Reply:
We would use the query steps to see the formats that the USB camera supports. We can word it like:

Follow these [steps](/How_to_query_usb_camera.md) to see the output formats supported by your USB camera.